src/components/Explorer/queryTemplate.js

```diff
@@ -1,3 +1,25 @@
+function templ(strings, value) {
+  if (value.length === 0) {
+    return '';
+  }
+  let r = 'AND (';
+  const str0 = strings[0];
+  const str1 = strings[1];
+
+  if (value instanceof Array) {
+    value.forEach((s, index) => {
```

Comment:
I suggesting building the interior using `.map` and then saving it to a variable.

At the end you can do `AND(${var})`